system/Common.php

```diff
@@ -563,6 +563,22 @@ function function_usable(string $functionName): bool
     }
 }
 
+if (! function_exists('solidus')) {
+    /**
+     * Generates the solidus character (`/`) depending on the HTML5 compatibility flag in `Config\DocTypes`
+     *
+     * @internal
+     */
+    function solidus(): string
```

Comment:
How about renaming to `_solidus()`?
HTML helper has `_get_uri()`.

Reply:
I propose to rename it to `_solidus()`.
1. This is not a normal helper function.
2. Now it does output a space and a solidus or empty string.